contrib/hello-world-react/.eslintrc

```diff
@@ -0,0 +1,30 @@
+{
+	"env": {
+		"browser": true,
+		"node": true,
+		"es6": true,
+		"jest": true,
+		"commonjs": true
+	},
+	"extends": "eslint:recommended",
```

Comment:
I think it is better to extend from `eslint:recommended` and `plugin:react/recommended` which basically makes [line 28](https://github.com/intuit/wasabi/pull/72/files#diff-42f4433a10d03d8372dbd9078124bb1dR28) obsolete and allows for Components to be named whatever you want.